nonebot-plugin-resolver/core/comment.py

```
@@ -0,0 +1,935 @@
+
+        nodes.append(MessageSegment.node_custom(
+            user_id="80000000",
+            nickname=nickname,
+            content=node_msg
+        ))
+    return nodes
+
+
+def format_comments_to_nodes(bot_id, comments: List[Dict[str, Any]], title: str, nickname: str) -> list:
+    """
+    格式化为文字合并转发包 (包含真实图片与表情 bytes 字节，自适应动静图)
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)